lib/active_admin/orm/active_record/comments.rb

```diff
@@ -5,9 +5,9 @@
 
 # Add the comments configuration
 ActiveAdmin::Application.inheritable_setting :comments,                   true
-ActiveAdmin::Application.inheritable_setting :show_comments_in_menu,      true
 ActiveAdmin::Application.inheritable_setting :comments_registration_name, 'Comment'
 ActiveAdmin::Application.inheritable_setting :comments_order,             "created_at ASC"
+ActiveAdmin::Application.inheritable_setting :comments_menu,              false
```

Comment:
Ah, sorry - not sure why I changed it.
Picking up on the existing logic from @whitehat101 - would using `{}` as a default work for you? The current logic would except if `true` was input. Otherwise, I can make some changes to handle `true`, `false`, and `{ ... }`